.mergify.yml

```diff
@@ -50,6 +51,33 @@ pull_request_rules:
       - status-success~=AWS CodeBuild us-east-1
       - status-success=continuous-integration/travis-ci/pr
       - status-success=Semantic Pull Request
+      
+  - name: Synchronize that PR to upstream and merge it (no-squash)
+    actions:
+      dismiss_reviews:
+        approved: true
+        changes_requested: false
+      merge:
+        strict: smart
+        method: merge
+        strict_method: merge
+    conditions:
+      - author!=dependabot[bot]
+      - author!=dependabot-preview[bot]
+      - -title~=(WIP|wip)
+      - label!=blocked
+      - label!=do-not-merge
+      - label=pr-status/ready-to-merge
+      - label=no-squash
```

Comment:
Is it `no-squash` or `pr-status/no-squash`?

In the other repo we do a `label~=~` for substring matching.

Reply:
Yes, `pr-status/no-squash`. I will update.

Also, mergify docs say `=` should be sufficient to match against one label from the array.  `~=` is for regular expressions.